### PR TITLE
added the ability to add resolution when closing issue

### PR DESCRIPTION
**Pull request by JasBeilin**
## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: [link to the issue](https://jira-dc.paloaltonetworks.com/browse/CIAC-8934)

## Description
Adding option to resolve issue with specific resolution.

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

File: Packs/Jira/ReleaseNotes/3_1_5.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+
+#### Integrations
+
+##### Atlassian Jira v2 (Deprecated)
+
+- Added the **resolution** argument when closing an issue.
```
Comment (BEAdi) [inline Packs/Jira/ReleaseNotes/3_1_5.md:6]:
```suggestion
- Added the **resolution** argument to the ***jira-edit-issue*** command when closing an issue.
```

File: Packs/Jira/Integrations/JiraV2/JiraV2.yml [hunk 2]
```diff
@@ -402,6 +402,8 @@ script:
     - description: The component names of the issue. For example components="component1,component2".
       isArray: true
       name: components
+    - description: Resolution to change in the issue, to be used when transitioning to a closing status.
```
Comment (BEAdi) [inline Packs/Jira/Integrations/JiraV2/JiraV2.yml:405]:
```suggestion
    - description: Resolution to update in the issue, to be used when transitioning to a closed status.
```

**Review by BEAdi — APPROVED** [review]
Why is it added to Jira V2 and not V3?

**Comment by JasBeilin** [comment]
> Why is it added to Jira V2 and not V3?

Because we need it only for gold as the new Jira does not add Fixed resolution by default and in the V3 we handle the status change differently, It did not come from clients request.

**Pull request merged**